app/graphql/types/profile.rb

```diff
@@ -2,18 +2,19 @@ class Types::Profile < Types::BaseObject
   description 'A user profile on Kitsu'
 
   field :id, ID, null: false
+
   field :slug, String,
     null: true,
     description: 'The URL-friendly identifier for this profile'
-  field :url, String,
-    null: true,
-    description: 'The URL for this profile'
 
-  # Name
   field :name, String,
     null: false,
     description: 'A non-unique, user-visible name for the profile.  Can contain spaces, emoji, etc.'
 
+  field :url, String,
+    null: true,
+    description: 'The full URL for this profile'
+
```

Comment:
This was intentionally placed next to slug and ID since they're related, please put it back